tests/basic/customer_issues.go

```diff
@@ -0,0 +1,350 @@
+				command = fmt.Sprintf("iptables -D OUTPUT -p tcp -s %s -j DROP", eachIp)
+			}
+			log.InfoD("Triggering command [%s] from Node [%v]", command, eachNode.Name)
+			err := runCommand(command, eachNode)
+			if err != nil {
+				return err
+			}
+		}
+	}
+	return nil
+}
+
+// flushIptableRules flushes all IPtable rules on the node specified
+func flushIptableRules(n node.Node) error {
+	command := "iptables -F"
+	if !node.IsMasterNode(n) {
+		err := runCommand(command, n)
+		if err != nil {
+			return err
+		}
+		return nil
+	}
+	return nil
+}
+
+func getVolumeRuntimeState(vol string) (string, error) {
+	volDetails, err := Inst().V.InspectVolume(vol)
+	if err != nil {
+		return "", err
+	}
+	var runTimeStat string
+	runTimeStat = ""
+	log.InfoD("VolumeDetails [%v]", volDetails)
+	for _, v := range volDetails.RuntimeState {
+		log.InfoD("RuntimeState is in state %v", v)
+		runTimeStat = v.GetRuntimeState()["RuntimeState"]
+	}
+	return runTimeStat, nil
+}
+
+var _ = Describe("{FordRunFlatResync}", func() {
+	/*
+		Test Needs 10 VM's running with Internal KVDB
+		Cluster should have 6 StorageNodes and 4 Storageless nodes
+		App Used Mongodb on multiple Namespace would be running on both storage and storage-less nodes
+		Set the IPtables rules in some nodes  of that subnet such that they will not be reachable from all nodes in second subnet
+		Make sure cluster is up and IOs are running for MongoDB apps. Most of the volumes will go into degraded state
+		Wait for around 30 minutes to hours for more IOs.
+		Now set IPtables rules in another subnet
+		Remove the Iptables rules from all  nodes in subnet 1 and wait for them to join the cluster back.
+		Slowly remove IPtables rules from subnet2 nodes such that some volumes
+		 	will be in sync and IO started running. Now immediately block it again on few nodes so
+			that volume state will set to ‘NOT in Quorum’
+		Once this state is set then kill the etcd process in leader KVDB node and remove all IPtable rules from all nodes.
+			and set the IPtable rule in this node (where killed the etcd process)
+			so that it is not reachable to other nodes in subnet1
+		Wait for three copies to be created for KVDB and cluster to be up.
+		Now remove the IPtables rules from a node and wait for it to join.
+	*/
+	JustBeforeEach(func() {
+		StartTorpedoTest("FordRunFlatResync",
+			"Ford customer issue for runflat and resync failed PTX-16727",
+			nil, 0)
+	})
+	var contexts []*scheduler.Context
+	stepLog := "Ford customer issue for runflat and resync failed"
+	It(stepLog, func() {
+		var iptablesflushed bool
+		iptablesflushed = false
+
+		// Check if the cluster consists of 10 nodes
+		log.InfoD("Get all nodes present in the cluster")
+		allNodes := []node.Node{}
+		// create an array with storage and storage less nodes added
+		for _, each := range node.GetStorageDriverNodes() {
+			allNodes = append(allNodes, each)
+		}
+
+		for _, each := range node.GetStorageLessNodes() {
+			allNodes = append(allNodes, each)
```

Comment:
you can remove this as GetStorageDriverNodes() will include storageless nodes also